rn/Teacher/ios/Teacher/SpeedGrader/Drawer.swift

```diff
@@ -40,7 +43,7 @@ struct Drawer<Content: View>: View {
         self.content = content()
         self.minHeight = minHeight
         self.maxHeight = maxHeight
-        self._state = state
+        self._state = state.transaction(DrawerState.transaction)
```

Comment:
whenever this component updates this binding, it will use the drawer animation & disable any children (like the tab pages) animations that would make it look weird.